docs/sources/administration/cli.md

```diff
@@ -10,41 +10,215 @@ weight = 8
+```
+
+### Override homepath value  
+
+Sets the path for the Grafana install/home path, defaults to working directory. You do not need to use this if you are in the Grafana installation directory when using the CLI.
+ 
+**Example:**
+```bash
+grafana-cli admin reset-admin-password --homepath "c:\Program Files\grafana" mynewpassword
+```
+
+### Override config file         
+
+`--config value` overrides the default location where Grafana expects the configuration file. Refer to [Configuration]({{< relref "configuration.md" >}}) for more information about configuring Grafana and default configuration file locations.
+
+**Example:**
+```bash
+grafana-cli admin reset-admin-password --config "/etc/configuration/" mynewpassword
+```
+
+## Plugins commands
+
+Grafana CLI allows you to install, upgrade, and manage your Grafana plugins. For more information about installing plugins, refer to [plugins page]({{< relref "../plugins/installation.md" >}}).
+
+All listed commands apply to the Grafana default repositories and directories. You can override the defaults with [Global Options](#global-options).
+
+### List available plugins
+
+```bash
+grafana-cli plugins list-remote
+```
+
+### Install the latest version of a plugin
+
+```bash
+grafana-cli plugins install <plugin-id>
+```
+
+### Install a specific version of a plugin
+
+```bash
+grafana-cli plugins install <plugin-id> <version>
+```
 
-To show all admin commands:
-`grafana-cli admin`
+### List installed plugins
+
+```bash
+grafana-cli plugins ls
+```
+
+### Update all installed plugins
+```bash
+grafana-cli plugins update-all
+```
+
+### Update one plugin
+
+```bash
+grafana-cli plugins update <plugin-id>
+```
+
+### Remove one plugin
+
+```bash
+grafana-cli plugins remove <plugin-id>
+```
+
+## Admin commands
+
+Admin commands are only available in Grafana 4.1 and later.
+
+### Show all admin commands
+
+```bash
+grafana-cli admin
+```
 
 ### Reset admin password
 
-You can reset the password for the admin user using the CLI. The use case for this command is when you have lost the admin password.
+`grafana-cli admin reset-admin-password <new password>` resets the password for the admin user using the CLI. You might need to do this if you lose the admin password.
+
+If there are two flags being used to set the homepath and the config file path, then running the command returns this error:
+
+> Could not find config defaults, make sure homepath command line parameter is set or working directory is homepath 
 
-`grafana-cli admin reset-admin-password ...`
+To correct this, use the `--homepath` global option to specify the Grafana default homepath for this command:
+
+```bash
+grafana-cli --homepath "/usr/share/grafana" admin reset-admin-password <new password>
+```
 
-If running the command returns this error:
+If you have not lost the admin password then it is better to set it in the Grafana UI. ((HOW DO I DO THIS? I CANNOT FIND INSTRUCTIONS IN THE DOCS, NOTHING OBVIOUS IN UI.))
```

Comment:
@daniellee? @torkelo? Can you answer this?

Reply:
There is a change password tab in user profile page, and there is a password section in the server admin / users / user details page

Reply:
I found it 
![example](https://user-images.githubusercontent.com/52059945/71837441-45dd8200-306b-11ea-9c6d-18e8dd90c747.png)

Reply:
that is not the server admin / users / user details page but the organization user list. Have click the shield icon to go the server admin section

Reply:
Got it! Thanks, @torkelo. Change incoming shortly.